scss/exercisePreviewCard.module.scss

```diff
@@ -0,0 +1,35 @@
+@use './_variables.module';
+
+%exercisePreviewCard__topBorder {
```

Comment:
What's this `%`?

Reply:
It marks a block of code that can be extended with `@extend`
It apparently uses up less space in the compiled CSS compared to mixins because mixins will inline the CSS properties, but these will make a reused CSS class.
Look at the "Extend/Inheritance" section on this page: https://sass-lang.com/guide